Assets/Scripts/Audio/AudioController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/** The master audio manager--loads banks and 
+ contains functions that will start, stop, pause, resume sound events*/
+    
+public class AudioController : MonoBehaviour {
+
+    uint bankID;
```

Comment:
I'm getting a warning that bankID is never used. Do we need it?

Reply:
One of the methods requires me to have a "bankID," and I was contemplating whether I should remove it or not, but I wanted to make that final determination after I do more testing with new sound events other than the incinerator.